area/midgaard.are

```diff
@@ -5657,7 +5732,7 @@ The concourse heads south, passing by a beautiful park.
 S
 #3271
 On the Concourse near the park~
-You are walking down the Concoruse, along the western wall of Midgaard.  To
+You are walking down the Concourse, along the western wall of Midgaard.  To
```

Comment:
hahah nice spot. in other projects I've integrated a spelling checker...probably worth doing something similar here :)